docs/install/install-eventing-with-yaml.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Installing Knative Eventing using YAML files"
+linkTitle: "Install Eventing using YAML"
+weight: 03
+type: "docs"
+showlandingtoc: "false"
+---
+
+This topic describes how to install Knative Eventing.
+You can install the Knative Eventing component by applying YAML files using the `kubectl` CLI.
+
+When installing Knative, you can install the Serving component, Eventing component, or both on your cluster.
+For how to install the Serving component, see [Installing Serving using YAML files](./install-serving-with-yaml.md)
+
+## Prerequisites
+
+Before installation, you must meet the prerequisites.
+See [Knative Prerequisites](./prerequisites.md).
+
+
+## Install the Eventing component
+
+To install the Eventing component:
+
+1. Install the required custom resources:
+
+   ```bash
+   kubectl apply -f {{< artifact repo="eventing" file="eventing-crds.yaml" >}}
+   ```
+
+1. Install the core components of Eventing:
+
+   ```bash
+   kubectl apply -f {{< artifact repo="eventing" file="eventing-core.yaml" >}}
+   ```
+
+
+## Verify the installation
+
+Monitor the Knative components until all of the components show a `STATUS` of `Running`:
+
+```bash
+kubectl get pods --namespace knative-eventing
+```
+
+
+## Optional: Install a default Channel (messaging) layer
+
+The tabs below expand to show instructions for installing a default Channel (messaging) layer.
```

Comment:
I'd lowercase channel, also let's not use parenthesis where we can avoid it maybe? I don't know that we need to say messaging after channel every time.

Reply:
Just did a find and replace for all the uppercase occurrences of channel. It seems pretty consistently uppercase in this topic but it makes more sense to me for it to be lowercase as you suggest. The same goes for broker. I'll update them all.